app/cms/github.server.ts

```diff
@@ -325,7 +325,7 @@ export async function downloadFileByPath(
   const { data } = await octokit.repos.getContent({
     owner: source.owner,
     repo: source.name,
-    headers: { branch: source.branch },
+    ref: source.branch,
```

Comment:
@joekrill can you please sanity check that this is correct? i don't know where the headers thing came from

Reply:
I could swear I found this documented somewhere and I remember making this change. But I'm looking for it again and I can't find it anywhere. So I must have misread the docs or something.